alex/tools/kaldi/env_voip_cs.sh

```diff
@@ -39,5 +39,8 @@ export train_mmi_boost=0.05
 export mmi_beam=16.0
 export mmi_lat_beam=10.0
 
+export smbr_beam=16.0
+export smbr_lat_beam=10.0
```

Comment:
NOT very important: lattice-beam value of 10 is quite large which may be too slow and not necessary value around 6 should be enough